modules/getting-started/proc-techdocs-config-cicd-prep-repo.adoc

```diff
@@ -0,0 +1,12 @@
+[id="proc-techdocs-config-cicd-prep-repo_{context}"]
+
+= Preparing your repository for continuous iteration (CI)
```

Comment:
```suggestion
// Module included in the following assemblies:
//
// * assemblies/assembly-techdocs-plugin.adoc

:_mod-docs-content-type: PROCEDURE
[id="proc-techdocs-config-cicd-prep-repo_{context}"]
= Preparing your repository for CI
```

I think we should be consistent with use of CI in headings - in the earlier module, we have "CI/CD" without explaining it in parentheses, so is it really necessary to do that here? Alternatively maybe include an introduction line / abstract that explains what this procedure is for/doing where we can mention it?